powertools-logging/powertools-logging-logback/src/main/java/software/amazon/lambda/powertools/logging/internal/JsonUtils.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates.
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package software.amazon.lambda.powertools.logging.internal;
+
+/**
+ * Json tools to serialize attributes manually, to avoid using further dependencies (jackson, gson...)
+ */
+public class JsonUtils {
```

Comment:
`mvn dependency:tree` reveals a dependency from both this project and `powertools-logging` on `jackson-core` - is this necessary?

Reply:
The problem is that `powertools-logging` uses `powertools-serialization` which use jackson.
We use jackson to actually serialize the event in json in order to log it...
I agree that does not make sense to have this JsonUtils to avoid jackson if we have jackson anyway...

Reply:
The thing is that we serialize a json "manually", it's not an Object that is serialized automatically... I prefer to keep that to have control on what is generated

Reply:
Because there is no "Object" per se to serialize, we serialize progressively based on the required fields / available data in the event log. If you look at the base code of log4j, they do the same.